homeassistant/components/pushbullet/sensor.py

```diff
@@ -45,21 +83,24 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
 
     pbprovider = PushBulletNotificationProvider(pushbullet)
 
-    devices = []
-    for sensor_type in config[CONF_MONITORED_CONDITIONS]:
-        devices.append(PushBulletNotificationSensor(pbprovider, sensor_type))
-    add_entities(devices)
+    monitored_conditions = config[CONF_MONITORED_CONDITIONS]
+    entities = [
+        PushBulletNotificationSensor(pbprovider, description)
+        for description in SENSOR_TYPES
+        if description.key in monitored_conditions
+    ]
+    add_entities(entities)
 
 
 class PushBulletNotificationSensor(SensorEntity):
     """Representation of a Pushbullet Sensor."""
 
-    def __init__(self, pb, element):
+    def __init__(self, pb, description: SensorEntityDescription):
         """Initialize the Pushbullet sensor."""
+        self.entity_description = description
         self.pushbullet = pb
-        self._element = element
-        self._state = None
-        self._state_attributes = None
+
+        self._attr_name = f"Pushbullet {description.key}"
```

Comment:
The old code used the `key`. However `description.name` would probably make a bit more sense here.

Reply:
I agree, however, let's keep the impact of these changes as low as possible for now.